cmake/projects/utf8/hunter.cmake

```diff
@@ -19,6 +19,17 @@ hunter_add_version(
     438b70b0c6ee3f674e068ebbc19a4d04a5a4ca56
 )
 
+hunter_add_version(
+    PACKAGE_NAME
+    utf8
+    VERSION
+    2.3.4-p0
```

Comment:
> https://ci.appveyor.com/project/sjdrc/hunter/builds/23849747

You're trying to add `2.3.4-p0` and this is a test for `2.3.4-p1`, see log:
```
-- verifying file...
       file='C:/projects/hunter/_testing/Hunter/_Base/Download/utf8/2.3.4-p1/438b70b/v2.3.4-p1.tar.gz'
-- File already exists and hash match (skip download):
  file='C:/projects/hunter/_testing/Hunter/_Base/Download/utf8/2.3.4-p1/438b70b/v2.3.4-p1.tar.gz'
  SHA1='438b70b0c6ee3f674e068ebbc19a4d04a5a4ca56'
-- extracting...
     src='C:/projects/hunter/_testing/Hunter/_Base/Download/utf8/2.3.4-p1/438b70b/v2.3.4-p1.tar.gz'
     dst='C:/projects/hunter/_testing/Hunter/_Base/fb66098/cbfedc7/ef8f497/Build/utf8/Source'
-- extracting... [tar xfz]
-- extracting... [analysis]
-- extracting... [rename]
-- extracting... [clean up]
-- extracting... done
```

Actually I don't understand why do we need `*-p0`. We have fresh `*-p1` already in system.

Reply:
We need utf8 header to be installed to the include directory for compatibility with existing code. As there was already a release in the hunter fork, I thought it would make sense to simply use that.

Reply:
> As there was already a release in the hunter fork, I thought it would make sense to simply use that.

Okay, so you want to do `hunter_config(utf8 VERSION 2.3.4-p0)`.

What I was talking before that it will be probably better to add an option to `utf8` and release a new version. In this case, you can do:
```cmake
hunter_config(utf8 VERSION ${HUNTER_utf8_VERSION} CMAKE_ARGS UTF8_LEGACY_HEADER=ON)
```

It will be easier for you to do an update if a new `utf8` version will be released.